Maestro/subscriptions.json

```diff
@@ -102,6 +108,12 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 7644
     },
+    // A build definition that will update the dependencies of the dotnet/toolset repo
```

Comment:
I assume this will be used in the future?  I don't see it used here.

Reply:
Yes, I'm still working on updating the update-dependencies process in dotnet/toolset.  When that is done then we can add a subscription here.